driver-core/pom.xml

```diff
@@ -201,7 +201,7 @@
                         <_include>-osgi.bnd</_include>
                         <Import-Package>
                             <!-- JNR does not provide OSGi bundles, so exclude it; the driver can live without it -->
-                            <![CDATA[com.google.common*;version="[14.0,20)",!jnr.*,*]]></Import-Package>
+                            <![CDATA[com.google.common*;version="[14.0,20)",!jnr.*,!sun.misc.*,*]]></Import-Package>
```

Comment:
Thanks, I figured that the solution I came up with wasn't right.  I agree that it makes sense to do that, but it also seems weird to have the bundle import `sun.misc`, when it won't be used in every case, but we already need to deal with this as you mention for the metrics library.